department-of-veterans-affairs/abd-vro
# Bump ruby in svc-bgs-api/src/docker and create integration test

Bumps ruby from 2.7.8-alpine3.16 to 3.2.2-alpine3.16. Adds Integration Test

## What was the problem?

Not all logging statements were printed in docker container.
There is no integration test to test svc-bie-api
Associated tickets or Slack threads:
- Duplicates #1740

## How does this fix it?[^1]

Fixes issue where logging statements are not printed in docker container.
Adds an integration test. This test will need to be updated to validate the response in the message payload in the step titled `Validate response in payload of message from RabbitMQ add-note-response queue`

## How to test this PR
1.  Follow steps to start platform base and API Gateway [here](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Docker-Compose#platform-base--api-gateway)
2. Run svc-bgs-api:
```
COMPOSE_PROFILES=svc ./gradlew :app:dockerComposeUp 
```

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
